### PR TITLE
Ensured that the name of assembly sequences is assembly_sequence_name…

### PR DESCRIPTION
**Pull request by dpopleton**
Swapped logic to use seq_region.name as each sequences name.
Stored the production name as an attribute

### CODE REVIEW

File: src/ensembl/production/metadata/updater/core.py [hunk 1]
```diff
@@ -542,7 +543,22 @@ def new_genebuild(self, species_id, meta_session, db_uri, source=None):
             attribute=sample_location_attribute,
         )
         genebuild_dataset_attributes.append(sample_location_param)
-
+        # Add the production name:
+        production_name_attribute = meta_session.query(Attribute).filter(
+            Attribute.name == "production.production_name").one_or_none()
+        if production_name_attribute is None:
```
Comment (marcoooo) [inline src/ensembl/production/metadata/updater/core.py:549]: Personally, I would advise not to automatically create this one, but just raise an error if the attribute does not exists.

**Review by marcoooo — CHANGES_REQUESTED** [review]

File: src/ensembl/production/metadata/updater/core.py [hunk 2]
```diff
@@ -333,6 +333,7 @@ def get_assembly_sequences(self, species_id, assembly):
                     accession_info[seq_region_name]["karyotype_rank"] = karyotype_rank
             # Now, create AssemblySequence objects for each unique accession.
             for accession, info in accession_info.items():
+                seq_region_name = accession
```
Comment (marcoooo) [inline src/ensembl/production/metadata/updater/core.py:336]: Sorry I don't get the whole logic here: 
- seq_region_name <= accession
- then accession is updated with the first matching names in synonym ?
- `name = accession` is set but not used afterwards  
you'll need to explain this to me, thanks!

File: src/tests/test_updater.py [hunk 3]
```diff
@@ -52,7 +52,7 @@ def test_new_organism(self, multi_dbs):
             assert dataset.dataset_source.type == "core"
             assert dataset.dataset_type.name == "genebuild"
 
-    #
+    # Just this one!.
```
Comment (marcoooo) [inline src/tests/test_updater.py:55]: not sure about the meaning of this comment :-)

File: src/ensembl/production/metadata/api/models/utils.py [hunk 4]
```diff
@@ -22,6 +22,6 @@ def check_release_status(meta_dbc, dataset_uuid):
         # Now we check if there exists a genome dataset with the corresponding dataset_id and a non-null release_id
         result = session.query(
             session.query(GenomeDataset).filter(GenomeDataset.dataset_id == dataset_id,
-                                                GenomeDataset.ensembl_release is not None).exists()
+                                                GenomeDataset.ensembl_release != None).exists()
```
Comment (marcoooo) [inline src/ensembl/production/metadata/api/models/utils.py:25]: @bilalebi are you using this method is service?

File: src/tests/test_updater.py (hunk 3 above)
Comment (dpopleton) [reply src/tests/test_updater.py:55]: Shouldn't have been left in.

File: src/ensembl/production/metadata/api/models/utils.py (hunk 4 above)
Comment (bilalebi) [reply src/ensembl/production/metadata/api/models/utils.py:25]: No, I'm not, @marcoooo

**Review by marcoooo — APPROVED** [review]

**Pull request merged**